docs/payload-migration/10-cms-switching-considerations.md

```diff
@@ -0,0 +1,676 @@
+# Chapter 10: CMS Switching Considerations
```

Comment:
We aren't on Sanity officially yet. The current MYSQL/ PHP site is the source of truth and where we would originate all data to be migrated.